src/main/java/com/rental/rentalApp/repositories/VehicleRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.rental.rentalApp.repositories;
+
+import com.rental.rentalApp.entities.Vehicle;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+
+public interface VehicleRepository extends CrudRepository<Vehicle, Integer> {
+//    List<Vehicle> findAllVehicles();
```

Comment:
Just as a mock function for now

```suggestion
    List<Vehicle> findByVIN();
```